src/Tizen.Sensor/Tizen.Sensor/BatchData/BatchData.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Tizen.Sensor
+{
+    /// <summary>
+    /// The BatchData class is used for storing value of particular batch type sensor.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    public abstract class BatchData
+    {
+        /// <summary>
+        /// Initializes a new instance of the BatchData class.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// <param name="timeStamp">Timestamp of sensor event.</param>
```

Comment:
Method/delegate has an error in the documentation of parameter(s): Parameter 'timestamp' has a wrong name in the documentation: 'timeStamp'.